inst/builtin/bs5/shiny/_rules.scss

```diff
@@ -303,6 +308,10 @@ $icon-classes: ("bi", "fa", "fas", "far", "fab", "material-icons") !default;
 }
 
 @include color-mode(dark) {
+  .card {
+    --card-border-color: rgba(255, 255, 255, 0.1);
+  }
+
```

Comment:
Oops, this isn't needed. I added this before I found `$border-color-translucent-dark`

```suggestion
```